src/styles/styles.js

```diff
@@ -478,6 +476,62 @@ const styles = {
         marginRight: 8,
     },
 
+    expensiTextInputWrapper: {
+        height: 52,
+    },
+    expensiTextInputContainer: {
+        flex: 1,
+        borderWidth: 1,
+        borderRadius: variables.componentBorderRadiusNormal,
+        borderColor: themeColors.border,
+        paddingTop: 25,
+        paddingHorizontal: 12,
+        paddingVertical: 8,
+        justifyContent: 'center',
+        height: '100%',
+        backgroundColor: '#fff',
+    },
+    expensiTextInputContainerWithoutLabel: {
+        paddingTop: 0,
+        paddingBottom: 0,
+    },
+    expensiTextInputContainerOnFocus: {
+        borderColor: themeColors.borderFocus,
+    },
+    expensiTextInputContainerOnError: {
+        borderColor: themeColors.badgeDangerBG,
+    },
+    expensiTextInputLabel: {
+        position: 'absolute',
+        left: 12,
+        top: 14,
+        fontSize: variables.fontSizeNormal,
+        color: themeColors.textSupporting,
+        fontFamily: fontFamily.GTA,
+        width: '100%',
+    },
+    expensiTextInputLabelDesktop: {
+        transformOrigin: 'left center',
+    },
+    expensiTextInputLabelTransformation: (translateY, translateX, scale) => ({
+        transform: [
+            {translateY},
+            {translateX},
+            {scale},
+        ],
+    }),
+    expensiTextInput: {
```

Comment:
I'm 99% sure that there's an existing reusable text style that could be used instead of introducing this one. If there's really not, this style should be named something much more generic.

Reply:
I couldn't find any style to reuse it